airflow/executors/celery_executor.py

```diff
@@ -377,6 +384,52 @@ def _check_for_stalled_adopted_tasks(self):
             for key in timedout_keys:
                 self.change_state(key, State.FAILED)
 
+    @provide_session
+    def _clear_stuck_queued_tasks(self, session=None):
```

Comment:
```suggestion
    def _clear_stuck_queued_tasks(self: Session = NEW_SESSION) -> None:
```

while we are at it and since it is a new method